nixos/modules/services/blockchain/tezos/common.nix

```diff
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
```

Comment:
```suggestion
```

Reply:
This whole file is copied from Serokell, do I need their permission to remove the license header? cc @balsoft

Reply:
The terms of [the license](https://github.com/serokell/tezos-packaging/blob/master/LICENSES/LicenseRef-MIT-TQ.txt) require you to do two things: when copying the code, include the original copyright notice and the permission notice. The nixpkgs repository is licensed under MIT, so the “permissions notice” part is covered, however you still do need to include the copyright notice. Frankly, the easiest way to comply with the license would be to keep the [REUSE](https://reuse.software/) header in the file – that’s what it is there for.

If you want, you can move the copyright notice elsewhere, but it has to be in the nixpkgs repository (preferably, in some way that makes it clear that it applies only to these files). If you want to remove the notice entirely, then you need permission not from Serokell, but from the copyright holder, which is TQ in this case.

Reply:
So, is it OK to just keep these headers that I already included? I'm clearly not a lawyer 😅